conf.d/forgit.plugin.fish

```diff
@@ -3,7 +3,7 @@
 set INSTALL_DIR (dirname (dirname (status -f)))
 set -x FORGIT_INSTALL_DIR "$INSTALL_DIR/conf.d"
 set -x FORGIT "$FORGIT_INSTALL_DIR/bin/git-forgit"
-if [ ! -e "$FORGIT" ]
+if not test -e "$FORGIT"
```

Comment:
Can you break this into a separate review, it's a great change but I want to be as focused as possible per review.

Reply:
I agree it's small, but considering there is a potentially breaking change in the rest of this, having this split apart makes sure if we need to revert anything we don't lose this improvement! 

I would definitely prefer a separate review for this one (even if it's added overhead) I'll approve it ASAP. :)

Reply:
Since I find a few improvements to add, let me put it into a separated PR.